### ISSUE TITLE
Cannot import YAML in non-default namespace

### ISSUE THREAD

**Issue by aaronRancher** [rancher/dashboard#2587]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
Bring up a `v2.5.7` Rancher.
Create a downstream cluster.
From downstream cluster, create a new namespace.
From downstream cluster, navigate to cluster manager.
Import YAML into the new namespace.
![Screen Shot 2021-03-30 at 1 06 13 PM](https://user-images.githubusercontent.com/57409052/113049645-bb45de80-9158-11eb-906e-48d796ed6d40.png)
I used the following YAML to create a deployment.
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: testdeployment
  labels:
    app: testworkload
spec:
  replicas: 1
  selector:
    matchLabels:
      app: mysite
  template:
    metadata:
      labels:
        app: mysite
    spec:
      containers:
        - name: mysite
          image: sangeetha/mytestcontainer
          ports:
            - containerPort: 80
```

**Result:**
Import YAML fails with:
```
failed to create testnamespace/testdeployment apps/v1, Kind=Deployment for e0bd6332-3fa7-40ea-bbfc-e03739d6b524: namespaces "testnamespace" not found
```

**Other details that may be helpful:**
The user is able to import this YAML in the new namespace in Ember UI.

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): `v2.5.7`
- Installation option (single install/HA): Single install

**Comment by aaronRancher** [comment]
Error response:
```
{type: "error", links: {}, code: "ServerError",…}
code: "ServerError"
links: {}
message: "failed to create testnamespace/testdeployment apps/v1, Kind=Deployment for 184fe5c8-42d9-4464-b959-b23a92678789: namespaces "testnamespace" not found"
status: 500
type: "error"
```

**Comment by vincent99** [comment]
This is straight `kubectl apply`; if the namespace doesn't exist you have to create it first.

If you're doing this in <= 2.5.7, then it's only going to work in the `local` cluster because that's where the server applies the yaml (#2585). The UI is showing a list of namespaces from the currently selected cluster (because that's where the server is _supposed to_ apply it), then that namespace doesn't exist in `local` where it's actually being applied.

**Comment by aaronRancher** [comment]
Thank you, @vincent99. Will test this use case once the fix for #2585 is available.

**Comment by gaktive** [comment]
Based on the other YAML issue linked in rancher/dashboard#2585, this looks like it's fixed. I'll close this one for now.
